app/presenters/outcome_presenter.rb

```diff
@@ -10,45 +10,28 @@ def initialize(i18n_prefix, node, state = nil, options = {})
   end
 
   def title
-    if use_template? && title_erb_template_exists?
+    if title_erb_template_exists?
       render_erb_template
       title = @view.content_for(:title) || ''
       strip_leading_spaces(title.chomp)
-    else
-      translate!('title')
     end
   end
 
-  def translate!(subkey)
-    output = super(subkey)
-    if output
-      output.gsub!(/\+\[data_partial:(\w+):(\w+)\]/) do |match|
-        render_data_partial($1, $2)
-      end
-    end
-
-    output
-  end
-
   def body(html: true)
-    if use_template? && body_erb_template_exists?
+    if body_erb_template_exists?
       render_erb_template
       govspeak = @view.content_for(:body) || ''
       govspeak = strip_leading_spaces(govspeak.to_str)
       html ? GovspeakPresenter.new(govspeak).html : govspeak
-    else
-      super
```

Comment:
I think it's fine to not raise an exception for now, but I do question whether that use case is actually sensible.